blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -156,25 +154,4 @@ Changes
 Fixes
 =====
 
-- Fixed an issue in the admin-ui to no longer display all columns as being
-  generated columns in the table/column view section.
-
-- Fixed an issue introduced in CrateDB 4.0 resulting in dysfunctional disk-based
-  allocation thresholds.
-
-- Fixed an issue resulting in ``pg_catalog.pg_attribute.attnum`` and
-  ``information_schema.columns.ordinal_position`` being ``NULL`` on tables
-  created with CrateDB < 4.0.
-
-- Fail the storage engine if indexing on a replica shard fails after it was
```

Comment:
looks like this was not backported?

Reply:
yes, it is the part of the es backport story, bringing the change log entry back.